app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        fin_sum = 0
+        for car in cars:
+            fin_sum += self.calculate_washing_price(car)
+            self.wash_single_car(car)
+        return round(fin_sum, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            return round((car.comfort_class * (self.clean_power
+                         - car.clean_mark) * self.average_rating
+                         / self.distance_from_city_center), 1)
+        return 0
```

Comment:
Also i don't really like the style of your code, work on improving it, you can use `black`

Reply:
explain in more detail what exactly I need to change in my code style

Reply:
And what do you mean when you say: use black?

Reply:
https://github.com/psf/black
https://www.geeksforgeeks.org/python-code-formatting-using-black/